php/msmtprc

```diff
@@ -0,0 +1,10 @@
+account default
```

Comment:
Why is this needed?

Reply:
This file is used only by docker to allow emails through an SMTP server. Without this emails will not reach the destination.